test/bdd/features/activitypub.feature

```diff
@@ -112,6 +112,7 @@ Feature:
     Then the JSON path "type" of the response equals "CollectionPage"
     And the JSON path "items" of the response does not contain "${domain2IRI}"
 
+""" TODO: Enable this test when CIDs for ipfs and local CAS are identical
```

Comment:
You can tag this TODO as #344 so I know to come here later and re-enable it

Reply:
https://github.com/trustbloc/orb/issues/344#issuecomment-842548994